Rakefile

```diff
@@ -2,3 +2,52 @@ require 'simp/rake/pupmod/helpers'
 require 'puppet-strings/tasks'
 
 Simp::Rake::Pupmod::Helpers.new(File.dirname(__FILE__))
+
+# TEMPFIX: When puppet-lint is fixed, delete this line and everything below it!
+# ------------------------------------------------------------------------------
+ignore_file = 'manifests/rules/scanblock.pp'
+
+PuppetLint.configuration.ignore_paths += [ignore_file]
+
+task :test do
+  puts <<-EOF
+
+================================================================================
```

Comment:
Wat? 
:sweat_smile:

In case this looks like it's redefining the `:test` task—it's not.  It just appends a second action to `:test` that will run after the normal one.   So the order of events will be:

1. run `rake test` and see the tests run normally.
2. see the failwhale
3. think to yourself, _"By Jove, that's right!  I must check `puppet-lint` to see if that's fixed yet, so we might remove this unsightly twaddle!"_